packages/components/tree-select/__tests__/tree-select.test.tsx

```diff
@@ -247,6 +247,8 @@ describe('TreeSelect.vue', () => {
     expect(document.querySelector('.el-select-dropdown__empty')).toBeFalsy()
     tree.vm.filter('no match')
     await nextTick()
+    await nextTick()
+    await nextTick()
```

Comment:
The three nextTicks seem a bit redundant, maybe we can use [vi.waitFor()](https://vitest.dev/api/vi.html#vi-waitfor)